nemo/collections/tts/fastspeech/model.py

```diff
@@ -0,0 +1,231 @@
+import collections
```

Comment:
you missed header here.
Also, could you please move all pytorch classes under tts/parts ?